manifest.php

```diff
@@ -73,7 +73,8 @@
             InstallEnhancedDeliveryLog::class,
             RegisterRdsSyncLogStorage::class,
             \oat\taoSync\scripts\install\RegisterTestCenterService::class,
-            \oat\taoSync\scripts\install\RegisterResultDataServices::class
+            \oat\taoSync\scripts\install\RegisterResultDataServices::class,
+            \oat\taoSync\scripts\install\RegisterEntityChecksumCache::class
```

Comment:
We have to register this service only on VM, we don't need this functionality on the central server. I think `taoOffline/scripts/tools/setup/SetupClientServer.php` would be better place for that.

Reply:
Right! I've removed the setup from this extension and will create a backport PR in `taoOffline`.